docs/DreamCast_Specs.md

```diff
@@ -0,0 +1,22 @@
+# SEGA Dreamcast Specifications
```

Comment:
Why not name this "Dreamcast_Overview.md" to match the naomi one?

Reply:
Because in this doc there are only the Machine's specs, in Naomi there's a bit more info.